<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ecg_app</class>
 <widget class="QMainWindow" name="ecg_app">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ECG监护系统</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab_realtime">
       <attribute name="title">
        <string>实时监护</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_history">
       <attribute name="title">
        <string>历史查询</string>
       </attribute>
      </widget>
      <widget class="QWidget" name="tab_settings">
       <attribute name="title">
        <string>设置</string>
       </attribute>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>文件</string>
    </property>
    <addaction name="actionConnect"/>
    <addaction name="actionDisconnect"/>
    <addaction name="separator"/>
    <addaction name="actionExport"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuData">
    <property name="title">
     <string>数据</string>
    </property>
    <addaction name="actionSync"/>
    <addaction name="actionShare"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>帮助</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuData"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionConnect">
   <property name="text">
    <string>连接设备</string>
   </property>
  </action>
  <action name="actionDisconnect">
   <property name="text">
    <string>断开连接</string>
   </property>
  </action>
  <action name="actionExport">
   <property name="text">
    <string>导出数据</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>退出</string>
   </property>
  </action>
  <action name="actionSync">
   <property name="text">
    <string>同步到云端</string>
   </property>
  </action>
  <action name="actionShare">
   <property name="text">
    <string>分享数据</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="text">
    <string>关于</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>